OpenXmlPowerTools/HtmlToWmlConverterCore.cs

```diff
@@ -1232,55 +1231,28 @@ private static XElement FontMerge(XElement higherPriorityFont, XElement lowerPri
 
             try
             {
-                using (Font f = new Font(ff, (float)sz / 2f, fs))
-                {
-                    const TextFormatFlags tff = TextFormatFlags.NoPadding;
-                    var proposedSize = new Size(int.MaxValue, int.MaxValue);
-                    var sf = TextRenderer.MeasureText(runText, f, proposedSize, tff);
-                    // sf returns size in pixels
-                    return sf.Width / multiplier;
-                }
+                return MetricsGetter.GetTextWidth(ff, fs, sz, runText) / multiplier;
```

Comment:
Can you update this so that MetrixGetter.GetTextWidth always returns a value - any error checking is handled there?